dnf/artifact-registry.py

```diff
@@ -1,4 +1,4 @@
-#  Copyright 2021 Google LLC
+#  Copyright 2022 Google LLC
```

Comment:
Nit: don't update the year here and elsewhere

https://opensource.google/documentation/reference/copyright#the_year

You actually weaken the copyright assertion by updating this because it's the _earliest_ year of the copyrighted work.

Reply:
thanks, this was new info for me. updated in latest commit.